doc/installing.rst

```diff
@@ -112,6 +112,16 @@ Instructions - Windows 10
 
 - If using the Arduino IDE for Visual Studio (https://www.visualmicro.com/), be sure to click Tools - Visual Micro - Rescan Toolchains and Libraries 
 
+-  You may see some error messages about missing files related to the
```

Comment:
I think the instructions should go further up, right after Download binary tools. Also, I suggest rephrasing in a more straightforward manner, something along the lines of:

- Initialize the submodules:
cd blah
git submodule update --init

If error messages about missing files related to the SoftwareSerial lib are encountered, it's because .... blah ... To get rid of the error messages, this step is mandatory.